scripts/azureml-assets/azureml/assets/config.py

```diff
@@ -345,7 +351,73 @@ def __init__(self, branch: str, uri: str):
         self._branch = branch
         super().__init__(PathType.GIT, uri)
 
+class DataConfig(Config):

```

Comment:
Would you be willing to add some validation tests for this class under the /test directory?

Reply:
Yes Louie, will be adding test cases .